package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "with-typescript",
   "version": "1.0.0",
+  "license": "ISC",
```

Comment:
I run `npx sort-package-json` to sort alphabetically package.json. So nothing actually changed (except new mdx dependencies of course).